languages/wikipron/src/generalized_split.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+import regex
```

Comment:
Need to add this to `requirements.txt` before submitting---we can discuss offline?